sources/HUBCollectionView.m

```diff
@@ -42,6 +46,27 @@ - (void)setContentOffset:(CGPoint)contentOffset
     [super setContentOffset:contentOffset];
 }
 
+- (NSMutableSet<NSString *> *)registeredCollectionViewCellReuseIdentifiers
+{
+    if (!_registeredCollectionViewCellReuseIdentifiers) {
+        _registeredCollectionViewCellReuseIdentifiers = [NSMutableSet new];
+    }
+
+    return _registeredCollectionViewCellReuseIdentifiers;
+}
+
+- (__kindof UICollectionViewCell *)dequeueReusableCellWithReuseIdentifier:(NSString *)identifier
```

Comment:
I don't really have a strong opinion, but would it be better to just add this logic into an overridden `registerClass: forCellWithReuseIdentifier:`? That way we wouldn't have to add to the public API of this class.